services/core-api/app/api/projects/project_summary/resources/project_summary.py

```diff
@@ -308,9 +270,39 @@ def put(self, project_guid, project_summary_guid):
             data.get('contacts', []))
         project.save()
 
-        if len(data.get('documents')) > 0:
-            project = Project.find_by_project_guid(project_guid)
-            ProjectUtil.notifiy_file_updates(project, mine)
+        # notify of status changes
+        if prev_status != project_summary.status_code:
+            message = ''
+            extra_data = {'project': {'project_guid': str(project.project_guid)}}
+            if prev_status == 'DFT' and project_summary.status_code == 'SUB':
+                message = f'A new major project description for ({project.project_title}) has been submitted for ({project.mine_name})'
+
+            if project_summary.status_code == 'ASG':
```

Comment:
here I mostly just put all the repetitive stuff at the top/bottom (data at top, calls at bottom).
But the ASG status was missing.